skytap/config.go

```diff
@@ -24,6 +24,7 @@ type SkytapClient struct {
 	vmsClient               skytap.VMsService
 	interfacesClient        skytap.InterfacesService
 	publishedServicesClient skytap.PublishedServicesService
+	names                   interface{}
```

Comment:
Why we store names with the service with the provider configuration, should not be part of the state.

Reply:
This is going to be addressed as part of #18

Reply:
done